app/Http/Controllers/Api/EtatDesLieuxSyncController.php

```
@@ -0,0 +1,98 @@
+                $type = $operation['type'] ?? null;
+                $payload = $operation['payload'] ?? [];
+
+                if ($type === 'CREATE_REPORT') {
+                    $report = $service->createFromSync($user, $payload);
+                    if ($report) {
+                        $processed++;
+                    } else {
+                        $failed++;
+                    }
+                } elseif ($type === 'UPLOAD_PHOTO') {
+                    $report = RentalConditionReport::where('client_key', $payload['report_key'] ?? '')->first();
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Models\Locations\RentalConditionReport::where().


(staticMethod.notFound)